README.md

```diff
@@ -184,7 +184,8 @@ The first part of the flow is to read data from a source.
 This can be a database, a csv file, a json file, a text file, a website, a pdf, a word document, an excel file, ...
 The only requirement is that you can read the data and that you can extract the text from it.
 
-For now we only support text files, pdf and docx but we plan to support other data type in the future.
+For now we only support text files, PDF, DOCX and HTML. but we plan to support other data type in the future.
+To only supports HTML files, if you `composer require html2text/html2text`.
```

Comment:
I think we could add this library as a full dependency, what do you think?

Reply:
As you like?